src/exec.js

```diff
@@ -27,6 +27,17 @@ function execSyncRead(command, silent = false) {
   return _.trim(String(cp.execSync(normalized, { stdio: ['pipe', 'pipe', 'pipe'] })));
 }
 
+function execSyncReadStdout(command, silent = false) {
+  const normalized = normalizeSpace(command);
+  if (_.isEmpty(normalized)) {
+    return '';
+  }
+  if (!silent) {
+    console.log(normalized);
+  }
+  return _.trim(String(cp.execSync(normalized, { stdio: ['inherit', 'pipe', 'inherit'] })));
+}
+
```

Comment:
I can just add another parameter (`stdio`) for `execSyncRead`, with a default value (`pipe`)